core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -240,24 +232,27 @@ public X509Certificate getTrustedCertificateAuthority() throws CertificateExcept
      * @return The trust anchor or {@code null} if no trusted certificate authority
```

Comment:
It is usually more convenient to return an empty list instead of `null`. This makes code using this method easier to implement because there is no need for null checks on the returned result